airflow/providers/amazon/aws/operators/redshift_data.py

```diff
@@ -110,63 +103,64 @@ def __init__(
         self.statement_name = statement_name
         self.with_event = with_event
         self.wait_for_completion = wait_for_completion
-        if poll_interval > 0:
-            self.poll_interval = poll_interval
-        else:
-            self.log.warning(
-                "Invalid poll_interval:",
-                poll_interval,
-            )
+        self.poll_interval = max(poll_interval, 1)
         self.return_sql_result = return_sql_result
         self.statement_id: str | None = None
         self.deferrable = deferrable
 
     def execute(self, context: Context) -> GetStatementResultResponseTypeDef | str:
```

Comment:
We now return a list right? Same for `execute_complete`

```suggestion
    def execute(self, context: Context) -> list[GetStatementResultResponseTypeDef] | str:
```

Reply:
yes